NuRadioMC/SignalGen/parametrizations.py

```diff
@@ -68,13 +69,21 @@ def get_time_trace(energy, theta, N, dt, shower_type, n_index, R, model, seed=No
         if False, for each request a new random shower realization is choosen.
         if True, the shower from the last request of the same shower type is used. This is needed to get the Askaryan
         signal for both ray tracing solutions from the same shower.
+    k_L: None or float
+        the k_L parameter for EM showers of the Alvarez2009 model. If a this parameter is provided, this value is used
+        and the parameter will not be drawn from a random distribution. 
+        This setting overrides the `same_shower` setting
+    full_output: bool (default False)    
```

Comment:
The default is actually True. And some more docs would be nice what 'additiona' output is returned?

Reply:
good catch. I changed it and documented the option